src/target/tag.cc

```diff
@@ -21,9 +21,22 @@
  * \file src/target/target_tag.cc
  * \brief Target tag registry
  */
+
+#define TVM_REGISTER_CUDA_TAG(Name, Arch, SharedMem, RegPerBlock)\
+TVM_REGISTER_TARGET_TAG(Name)             \
+    .set_config({                                       \
+        {"kind", String("cuda")},                       \
+        {"arch", String(Arch)},                         \
+        {"shared_memory_per_block", Integer(SharedMem)},\
+        {"registers_per_block", Integer(RegPerBlock)},  \
+        {"max_threads_per_block", Integer(1024)},       \
+        {"thread_warp_size", Integer(32)},              \
+    });
+
```

Comment:
Please move the macro definition to immediately before `TVM_REGISTER_CUDA_TAG `, and undef it once it is used

Reply:
Fixed.